ui/public/i18n/it/translation.json

```diff
@@ -51,7 +51,10 @@
     "timezone_tooltip": "Imposta la timezone di default per questa installazione di NethVoice",
     "create_nethvoice_adm": "Creazione NethVoice Administrator per gestione utenti",
     "remove_nethvoice_adm": "Rimozione NethVoice Administrator per gestione utenti",
-    "set_nethvoice_adm_password": "Impostazione NethVoice Administrator password per gestione utenti"
+    "set_nethvoice_adm_password": "Impostazione NethVoice Administrator password per gestione utenti",
```

Comment:
Non-english translations should be done on Weblate, not editing the translations file directly